projects/1.18/assets/little-logistics/zh_cn.json

```
@@ -0,0 +1,104 @@
+{
+  "item.littlelogistics.tug": "蒸汽拖船",
+  "item.littlelogistics.barge": "货柜驳船",
+  "item.littlelogistics.chunk_loader_barge": "区块加载驳船",
+  "item.littlelogistics.seater_barge": "坐式驳船",
+  "item.littlelogistics.fishing_barge": "自动捕鱼驳船",
+  "item.littlelogistics.spring": "车链",
+  "item.littlelogistics.tug_route": "拖船航线",
+  "item.littlelogistics.locomotive_route": "机车路线",
+  "item.littlelogistics.energy_tug": "能源拖船",
```

```suggestion
  "item.littlelogistics.energy_tug": "电力拖船",
```
此处没有好翻译（）
下方`Energy`部分也应相应做出修改